src/test/java/com/accelad/docx4j/tags/TagScannerTest.java

```diff
@@ -200,6 +200,34 @@ public void should_create_tag_with_word_items_when_tag_identifier_is_composed_by
         assertThat(wordItemsActual, hasItems(wordItem2, wordItem3, wordItem4));
     }
 
+    @Test
+    public void should_create_tag_when_tag_identifier_is_divided_in_two_parts() {
+        Paragraphs paragraphs = new Paragraphs();
+        when(aParagraph.asString()).thenReturn(A_PARAGRAPH_WITH_TAG_NAME);
+        paragraphs.add(aParagraph);
+        ListWordItems wordItems = new ListWordItems();
+        Run wordItem1 = new Run();
+        wordItem1.addText(new Text("A paragraph "));
+        Run wordItem2 = new Run();
+        wordItem2.addText(new Text("${FIRST PART"));
+
+        Run wordItem4 = new Run();
```

Comment:
After 2 it's 3!! Maybe copy paste error!
More seriously, could we extract the construction of a run in a method? And probably refactor others tests at the same time. Or Run could have an other constructor with a list of Text, one Text or vararg Text?